lib/scholar/metrics/similarity.ex

```diff
@@ -57,13 +57,15 @@ defmodule Scholar.Metrics.Similarity do
   Calculates Jaccard similarity based on binary attributes.
 
   $$
-  J = \frac{M\_{11}}{M\_{01} + M\_{10} + M\_{11}}
+  J(X, Y) = \frac{M\_{11}}{M\_{01} + M\_{10} + M\_{11}}
```

Comment:
```suggestion
  J(X, Y) = \frac{M_{11}}{M_{01} + M_{10} + M_{11}}
```

There's no M_{00} in the actual formula. This looks incorrect

Reply:
https://en.wikipedia.org/wiki/Jaccard_index#Similarity_of_asymmetric_binary_attributes

Reply:
No need to mention M_00 below then